crates/bevy_animation/src/_experimental.rs

```diff
@@ -0,0 +1,142 @@
+//! Contains unused experimental ideas
```

Comment:
Could you gives us a hint on how experimental your experimentals are? So far I would see them as none functional but I would have to take a closer look on a checkout.

Reply:
This is just stuff that I was experimenting with nothing relevant at this stage;

Reply:
File removed